### PR TITLE
ROC-3303 Claim form pdf content revision

### PR DESCRIPTION
**Pull request by rattur**
Content changes to claim pdf and sealed copy

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-3303

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimContentProvider.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import uk.gov.hmcts.cmc.claimstore.services.staff.content.ClaimantContentProvider;
 import uk.gov.hmcts.cmc.claimstore.services.staff.content.PersonContentProvider;
 import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.otherparty.IndividualDetails;
```
Comment (timja-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimContentProvider.java]: resolve conflicts and please attach sample PDF

Comment (rattur) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimContentProvider.java]: [000MC031-sealed-claim.pdf](https://github.com/hmcts/cmc-claim-store/files/1820037/000MC031-sealed-claim.pdf)
[000MC029-claim-receipt (1).pdf](https://github.com/hmcts/cmc-claim-store/files/1820038/000MC029-claim-receipt.1.pdf)

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/ClaimantContentProvider.java [hunk 2]
```diff
@@ -23,15 +27,17 @@ public ClaimantContentProvider(PersonContentProvider personContentProvider) {
     public ClaimantContent createContent(Party claimant, String submitterEmail) {
         requireNonNull(claimant);
         requireNonBlank(submitterEmail);
+        LocalDate dateOfBirth = claimant instanceof Individual?((Individual)claimant).getDateOfBirth(): null;
```
Comment (timja-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/ClaimantContentProvider.java]: please add spacing around the `?` and `:` ternary operators

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/PersonContent.java [hunk 3]
```diff
@@ -57,4 +65,10 @@ public String getContactPerson() {
     public String getBusinessName() {
         return businessName;
     }
+
+    public String getMobileNumber() {
+        return mobileNumber;
+    }
+
+    public String getDateOfBirth() { return dateOfBirth; }
```
Comment (timja-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/PersonContent.java]: fix spacing please... new line etc

File: src/main/resources/citizen/templates/document/claimIssueReceipt.html [hunk 4]
```diff
@@ -368,17 +403,23 @@
           {{ claimant.fullName }}
         </td>
       </tr>
+      <tr>
+        <td></td>
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/claimIssueReceipt.html]: do you need to add an empty cell? could you use rowspan="2"?

File: src/main/resources/staff/templates/document/sealedClaim.html [hunk 5]
```diff
@@ -349,15 +388,15 @@
...
+    <strong>Statement of truth</strong>
   </div>
   <div class="section-body">
     <table>
       <tr>
         <td></td>
         {% if claim.signerName is empty %}
           <td>
-            I, {{ claimant.fullName }}, believe that the facts stated in this claim form are true.
+            I  believe that the facts stated in this claim form are true.
```
Comment (timja-kainos) [inline src/main/resources/staff/templates/document/sealedClaim.html]: looks like there is an extra space here

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/ClaimantContentProvider.java [hunk 6]
```diff
@@ -40,7 +46,16 @@ public ClaimantContent createContent(Party claimant, String submitterEmail) {
...
             submitterEmail,
             personContent.getContactPerson(),
-            personContent.getBusinessName()
+            personContent.getBusinessName(),
+            personContent.getMobileNumber(),
+            personContent.getDateOfBirth()
         );
     }
+
+    private Optional<LocalDate> claimantDateOfBirth(Party party) {
```
Comment (geek-soft) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/ClaimantContentProvider.java]: Please move it into `uk.gov.hmcts.cmc.domain.utils.PartyUtils`. That is the place we have similar logic.

Comment (rattur) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/ClaimantContentProvider.java]: Attached the latest claim pdf and sealed copy
[000MC047-claim-receipt.pdf](https://github.com/hmcts/cmc-claim-store/files/1826424/000MC047-claim-receipt.pdf)
[000MC047-sealed-claim.pdf](https://github.com/hmcts/cmc-claim-store/files/1826425/000MC047-sealed-claim.pdf)

File: src/main/resources/citizen/templates/document/claimIssueReceipt.html [hunk 7]
```diff
@@ -3,26 +3,50 @@
...
+        font-size: 11px;
+      }
     }
 
     html {
       /*noinspection CssNoGenericFontName*/
       font-family: "Open Sans";
     }
 
+    .padding-bottom {
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/claimIssueReceipt.html]: padding is an implementation detail, how about `spacing-after` ?

File: src/main/resources/citizen/templates/document/claimIssueReceipt.html [hunk 8]
```diff
@@ -245,59 +280,89 @@
...
 <div class="section">
   <div class="section-heading">
-    <strong>Claim amount</strong>
+    <strong>Claim</strong>
   </div>
   <div class="section-body">
     <table>
       <tr>
-        <td>Claim amount:</td>
+        <td colspan="2" style="white-space: pre-line"><strong>Reason for claim:</strong><br/>
```
[the remark this answers is not in the document]
Comment (rattur) [reply src/main/resources/citizen/templates/document/claimIssueReceipt.html]: Attached new claim pdf and sealed copy
[000MC049-claim-receipt.pdf](https://github.com/hmcts/cmc-claim-store/files/1829034/000MC049-claim-receipt.pdf)
[000MC049-sealed-claim.pdf](https://github.com/hmcts/cmc-claim-store/files/1829035/000MC049-sealed-claim.pdf)

**Comment by rattur** [comment]
These are the new pdf documents with content updates and file name changes
[000MC076-claim-form.pdf](https://github.com/hmcts/cmc-claim-store/files/1830248/000MC076-claim-form.pdf)
[000MC076-claim-form-claimant-copy.pdf](https://github.com/hmcts/cmc-claim-store/files/1830249/000MC076-claim-form-claimant-copy.pdf)

File: domain-model/src/test/java/uk/gov/hmcts/cmc/domain/utils/PartyUtilsTest.java [hunk 9]
```diff
@@ -128,4 +128,11 @@ public void getBusinessNameReturnsEmptyOptionalWhenNotSoleTraderType() {
             .isEqualTo(Optional.empty());
     }
 
+    @Test
+    public void getClaimantDateOfBirth() {
```
Comment (geek-soft) [inline domain-model/src/test/java/uk/gov/hmcts/cmc/domain/utils/PartyUtilsTest.java]: Would be great to have a use case for non individual but not a blocker.

**Comment by rattur** [comment]
Newly updated claim pdf and sealed copy
[000MC083-claim-form-claimant-copy.pdf](https://github.com/hmcts/cmc-claim-store/files/1837353/000MC083-claim-form-claimant-copy.pdf)
[000MC083-claim-form.pdf](https://github.com/hmcts/cmc-claim-store/files/1837354/000MC083-claim-form.pdf)

**Pull request merged**